src/skins/vector/css/common.css

```diff
@@ -264,6 +250,12 @@ input[type=text]:focus, textarea:focus {
     color: white;
 }
 
+.emojione {
```

Comment:
I could have sworn i renamed the `.emojione` class to actually fit in with the CSS naming scheme of the rest of the app.  Assuming this still refers to the "jumbo emoji" that appears when a line contains nothing but emoji, we should definitely rename it to be consistent rather than un-namespaced and ambiguous.